token-syncer/src/token_syncer/commands/syncer.clj

```diff
@@ -78,75 +78,71 @@
 
-                  (nil? latest-root)
-                  {:code :error/token-read
-                   :details {:message "token root missing from latest token description"}}
+                      (nil? latest-root)
+                      {:code :error/token-read
+                       :details {:message "token root missing from latest token description"}}
 
-                  (and latest-root
-                       (= latest-token-description description))
-                  {:code :success/token-match}
+                      (and latest-root
+                           (= latest-token-description description))
+                      {:code :success/token-match}
 
-                  ;; deleted on both clusters, irrespective of remaining values
-                  (and (get latest-token-description "deleted")
-                       (get description "deleted"))
-                  {:code :error/tokens-deleted
-                   :details {:message "soft-deleted tokens should have already been hard-deleted"}}
+                      ;; deleted on both clusters, irrespective of remaining values
+                      (and (get latest-token-description "deleted")
+                           (get description "deleted"))
+                      {:code :error/tokens-deleted
+                       :details {:message "soft-deleted tokens should have already been hard-deleted"}}
 
-                  (and (some? opt-out-metadata-name)
-                       ;; opt-out of syncing if any of the descriptions signal opt-out intent
-                       (or (= "true" (get-in latest-token-description ["metadata" opt-out-metadata-name]))
-                           (= "true" (get-in description ["metadata" opt-out-metadata-name]))))
-                  {:code :success/skip-opt-out}
+                      (and (some? opt-out-metadata-name)
+                           ;; opt-out of syncing if any of the descriptions signal opt-out intent
+                           (or (= "true" (get-in latest-token-description ["metadata" opt-out-metadata-name]))
+                               (= "true" (get-in description ["metadata" opt-out-metadata-name]))))
+                      {:code :success/skip-opt-out}
 
-                  ;; active token, content the same irrespective of system metadata keys but roots different
-                  (and (seq description)
-                       (not= latest-root cluster-root)
-                       (not (get latest-token-description "deleted"))
-                       (not (get description "deleted"))
-                       (= (apply dissoc latest-token-description system-metadata-keys)
-                          (apply dissoc description system-metadata-keys)))
-                  (if (= latest-update-user cluster-update-user)
-                    {:code :success/skip-token-sync}
-                    {:code :error/token-sync
-                     :details {:message "token contents match, but were edited by different users"}})
```

Comment:
We should identify and log this instance as we are wiping out history from the target cluster.

Reply:
I logged the situation in the portion where we attempt to sync the token